src/current/v23.2/build-a-java-app-with-cockroachdb.md

```diff
@@ -215,6 +206,48 @@ BUILD SUCCESSFUL in 8s
 
 ## Recommended Practices
 
+### Set session variables
+
+[Session variables]({% link {{ page.version.version }}/session-variables.md %}) can be set in the JDBC connection string or as properties of the JDBC data source.
+
+To set the session variable in the JDBC connection string, add them to the [`options` parameter]({% link {{ page.version.version }}/connection-parameters.md %}#supported-options-parameters):
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export JDBC_DATABASE_URL=jdbc:postgresql://{host}:{port}/{database}?options=-c%20{session variable name}%3D{session variable value}
+~~~
+
+Where:
+
+- `{session variable name}` is the name of the session variable.
+- `{session variable value}` is the value of the session variable.
+
+URL encode the `options` parameter to make sure the JDBC connection URL is parsed correctly. For example, the following URL encoded `options` parameter:
+
+~~~ shell
+export JDBC_DATABASE_URL=jdbc:postgres://localhost:26257/movr?options=-c%20unbounded_parallel_scans%3Don
+~~~
+
+is equivalent to:
+
+~~~
+options=-c unbounded_parallel_scans=on
+~~~
+
+To set session variables as properties of the JDBC data source, set `options` using `setProperty`:
+
+{% include_cached copy-clipboard.html %}
+~~~ java
+Properties props = new Properties();
+props.setProperty("options", "-c {session variable name}={session variable value}");
```

Comment:
I'd add an example with 2 variables else you won't know if you need to add a comma, an additional property, or a space... this is how you do it:

```
ds.addDataSourceProperty("options", "-c unbounded_parallel_scans=true -c statement_timeout=30");
```

Reply:
Done.